executor_bot/src/dex/raydium.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  TOKEN_PROGRAM_ID,
+  findProgramAddress,
+  LIQUIDITY_PROGRAM_ID_V4,
+} from "@raydium-io/raydium-sdk";
+import { PublicKey } from "@solana/web3.js";
+import { Market } from "@project-serum/serum";
+import { deriveVaultSigner } from "./openbook";
+
+/**
+ * RAYDIUM V4 SWAP ACCOUNT ORDER
+ * 0 - Raydium program ID
+ * 1 - amm_id
+ * 2 - amm_authority
+ * 3 - amm_open_orders
+ * 4 - amm_target_orders
+ * 5 - pool_coin_token_account
+ * 6 - pool_pc_token_account
+ * 7 - serum_program_id
+ * 8 - serum_market
+ * 9 - serum_bids
+ * 10 - serum_asks
+ * 11 - serum_event_queue
+ * 12 - serum_coin_vault_account
+ * 13 - serum_pc_vault_account
+ * 14 - serum_vault_signer
+ * 15 - user_source_token_account
+ * 16 - user_destination_token_account
+ * 17 - user_source_owner
+ * 18 - SPL Token Program
+ */
+export const raydiumTradeAccts = async (
```

Comment:
This looks very similar to the Poseidon package's `Raydium.tradeAccounts` method. The primary difference is it's not using the Raydium SDK's `Liquidity.getAssociatedPoolKeys` method. 

Can we move this to the package and DRY up the duplicate code?